### PR TITLE
Minimal Skeleton for Alpaka pixel reco

### PR DESCRIPTION
**Pull request by felicepantaleo**
A minimal skeleton for the Alpaka pixel reconstruction was created. 
Only the `Framework` and `DataFormats` were copied from the `cuda` folder.

### CODE REVIEW

File: src/alpaka/Makefile [hunk 1]
```diff
@@ -0,0 +1,286 @@
+TARGETS = alpaka
```
Comment (makortel) [inline src/alpaka/Makefile:1]: This file looks mostly a copy from the pixel-standalone. Do I guess correctly that this is a work in progress and will be updated to be part of the build system of this repo at some point soon?

**Comment by makortel** [comment]
Just to note that `DataFormats` and `Framework` are equal to `cuda` program, `AlpakaCore` is added, `plugin-SiPixelClusterizer` has minimal content, and `CondFormats` and `bin` are missing (as are other plugin directories than the clusterizer).

**Comment by makortel** [comment]
Merging anyway

File: src/alpaka/Makefile (hunk 1 above)
Comment (felicepantaleo) [reply src/alpaka/Makefile:1]: yes, precisely

**Comment by felicepantaleo** [comment]
> Just to note that `DataFormats` and `Framework` are equal to `cuda` program, `AlpakaCore` is added, `plugin-SiPixelClusterizer` has minimal content, and `CondFormats` and `bin` are missing (as are other plugin directories than the clusterizer).

I'm trying to follow the minimal approach of adding things only when needed. Now we are working on the clusterizer. As soon as we have something else we will add the other folders

**Pull request merged**